packages/hoppscotch-common/src/components/lenses/renderers/JSONLensRenderer.vue

```diff
@@ -86,7 +86,14 @@
         />
       </div>
     </div>
-    <div ref="jsonResponse" class="flex flex-col flex-1 h-auto h-full"></div>
+    <div
+      ref="jsonResponse"
+      class="flex flex-col flex-1 h-auto h-full"
+      :class="{
+        responseToggleOff: !toggleFilter,
+        responseToggleOn: toggleFilter,
+      }"
```

Comment:
```suggestion
      :class="toggleFilter ? 'responseToggleOn' : 'responseToggleOff'"
```

How about leveraging ternary here?